piksi_tools/testing/Makefile

```diff
@@ -7,3 +6,5 @@ hub_ctrl: hub_ctrl.o
 clean:
 	rm -f *.o hub_ctrl
 
+usrp_test:
```

Comment:
I don't think this adds any value.  It doesn't build anything, and it's more obvious to just run the script than grep the makefile for the target.